demo_nodes_cpp/src/parameters/even_parameters_node.cpp

```diff
@@ -84,8 +80,16 @@ class EvenParameterNode : public rclcpp::Node
         }
         return result;
       };
-    this->set_on_parameters_set_callback(param_change_callback);
+    this->add_on_set_parameters_callback(param_change_callback);
   }
+
+  DEMO_NODES_CPP_PUBLIC
+  ~EvenParameterNode(void)
+  {
+    this->remove_on_set_parameters_callback(callback_handler.get());
+  }
```

Comment:
Destructor isn't necessary, the callback is automatically removed at node destruction.

Reply:
done